_posts/2023-06-16-deploy-on-kubernetes-with-helm.adoc

```diff
@@ -0,0 +1,400 @@
+
+## Install Helm and have access to a Kubernetes cluster.
+
+Please refer to https://helm.sh/docs/intro/install/[Installing Helm] page to install Helm in your environment.
+
+We will use https://kind.sigs.k8s.io/[kind] as our Kubernetes cluster.
+
+## Install Helm Repository for WildFly Chart
+
+The `wildfly` Chart can be installed from the https://docs.wildfly.org/wildfly-charts/ repository
+
+[source,nowrap]
+----
+$ helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+"wildfly" has been added to your repositories
+$ helm search repo wildfly
+NAME                  	CHART VERSION	APP VERSION	DESCRIPTION                                       
+wildfly/wildfly       	2.3.1        	           	Build and Deploy WildFly applications on OpenShift
+wildfly/wildfly-common	2.1.0        	           	A library chart for WildFly-based applications  
+----
+
+## Make sure your application pom contains the wildfly-maven-plugin plugin
+
+In our helloworld quickstart the wildfly-maven-plugin plugin is already present under the 'openshift' profile.
+Add it if you don't have it in your pom.xml.
+
+[source,nowrap]
+----
+<profiles>
+...
+        <profile>
+            <id>openshift</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <version>${version.wildfly.maven.plugin}</version>
+                        <configuration>
+                            <feature-packs>
+                                <feature-pack>
+                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                </feature-pack>
+                                <feature-pack>
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.cloud.fp}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <layer>cloud-server</layer>
+                            </layers>
+                            <filename>${project.artifactId}-${project.version}.war</filename>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+...
+    </profiles>    
+----
+
+Note: In our helloworld quickstart the maven-war-plugin is used to create the war. If you want to use a different plugin to create the war or jar you need to set the 'filename' configuration of the wildfly-maven-plugin plugin accordingly.
```

Comment:
I think you can remove this sentence completely, indeed I've realized there is no explicit `maven-war-plugin` declaration in the `helloworld/pom.xml`. In my opinion, this note here doesn't add value the the blog post goal

Reply:
I agree! Thanks.